.github/workflows/ci.yaml

```diff
@@ -38,6 +38,7 @@ jobs:
           - 5432:5432
         env:
           POSTGRES_PASSWORD: postgres
+          PGHOST: localhost
```

Comment:
I would invert the logics:
set PGHOST to localhost by default in the Makefile, and override this by setting oereb-db in docker-compose.yml